docs: update README with clearer explanations (#356)
threeport/threeport: docs/docs/architecture/threeport-core.md

![Threeport Stack](../img/ThreeportStack.png)

## Requirements

The following outline the primary requirements used to develop Threeport Core.

* Scalable: All components of Threeport Core are designed to scale to meet the global needs of an organization's software delivery.  The API, and all controllers are horizontally scalable.

> * Resilient: Threeport Core is designed to be installed across multiple regions as as to remain functional in the event of a regional outage.

Reviewer: ```suggestion
* Resilient: Threeport Core is designed to be installed across multiple regions to remain functional in the event of a regional outage.
```

> * Extensible: The core system is designed to accept module additions so that all common and specialized requirements can be accomodated in a single, coherent system.

Reviewer: ```suggestion
* Extensible: Threeport Core is extensible via modules so that all common and specialized requirements can be accommodated in a single, coherent system.
```

I think `extensible` is a keyword we should be sure to include when expanding on Threeport details. Also consolidating on `Threeport Core` instead of also saying `the core system`
